test/jit/test_functional_blocks.py

```diff
@@ -127,6 +128,59 @@ def test_intermediary_use():
         FileCheck().check_count("aten::add_", 1).run(graph)
         self.assertEqual(test_intermediary_use(), fn())
 
+    def test_remove_mutation_if_output(self):
+        class Mod(torch.nn.Module):
+            def __init__(self):
+                super(Mod, self).__init__()
+                self.mod = nn.Sequential(nn.Linear(20, 20), nn.ReLU(inplace=True))
+
+            def forward(self, x):
+                return self.mod(x)
+
+        with freeze_rng_state():
```

Comment:
why do we need to freeze rng state here? If that's important, should we just use `ones` instead of randn